src/js/flux.js

```diff
@@ -108,6 +109,26 @@ export var ArticleStore = DocBrown.createStore({
   listError: function(err) {
     this.setError(err, "list");
   },
+
+  import: function() {
+    this.resetError();
+    // We use the batch API mostly for testing purpose; we could simply append
+    // sample articles to the current list instead :)
+    var batch = this.api.createBatch();
+    return batch.createArticle.apply(batch, sampleArticles).process();
+  },
+
+  importSuccess: function(response) {
+    // XXX We want proper notification messages displayed to the end user here
+    response.responses.forEach(function(response) {
+      console.log("imported", response.body.title, "HTTP", response.status);
```

Comment:
The batch response status might well be always 200. Whereas the status of each response in its body could be 400, 409...